ironfish/src/utils/async.ts

```diff
@@ -11,6 +11,15 @@ export class AsyncUtils {
     return results
   }
 
+  static async *transform<TIn, TOut>(
+    iter: AsyncGenerator<TIn, unknown, unknown>,
```

Comment:
Can this take `AsyncIterable` instead? I think it's more general

Reply:
Changed